jayp-tech/VanierApp
# Implemented Task 2: added password toggle icon

## Purpose
Adding an eye icon next to password field to let users show or hide the password, making it easier to confirm that they are typing it correctly without compromising security.

## Screenshots
![image](https://github.com/user-attachments/assets/cc083336-7008-4b30-a9a5-10efbf50660b)

![image](https://github.com/user-attachments/assets/5c41756d-8225-4df0-95cf-cde516fa8c2a)

## Steps for QA
Check default behavior: Password is hidden (default).
Click the eye icon: Password should be visible.
Click again: Password should be hidden again.
